tests/test-war-smoke/pom.xml

```diff
@@ -251,12 +228,46 @@
                 <goals>
                   <goal>deploy</goal>
                 </goals>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>com.spotify</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>tag-image</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>tag</goal>
+                </goals>
+                <configuration>
+                  <image>jetty:${docker.tag.short}</image>
+                  <newName>${jetty.test.image}</newName>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>1.5.0</version>
+            <executions>
+              <execution>
+                <id>deploy-remote-docker</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
                 <configuration>
-                  <deploy.project>${app.deploy.project}</deploy.project>
-                  <deploy.version>${maven.build.timestamp}</deploy.version>
-                  <deploy.promote>false</deploy.promote>
-                  <dockerDirectory>${project.build.directory}/docker
-                  </dockerDirectory>
+                  <executable>gcloud</executable>
+                  <arguments>
+                    <argument>docker</argument>
+                    <argument>-q</argument>
+                    <argument>--</argument>
+                    <argument>push</argument>

```

Comment:
nope, that should prevent configuration making it into child pom's but I don't believe that applies for <executions/>

What would work here is matching the id of the exec plugin execution and then unbind the <phase>..it would match but not execute.  But...the issue there is that if you want it to push  in the parent and unbind in the child then you only ever push when running the parent, so you can't run just the build in the child and have image pushed.  Not a huge deal but if you want to see it work like that I'll just add a comment to that affect.